vars/getJenkinsLabels.groovy

```diff
@@ -12,7 +12,8 @@ def call(String backend, String aws_region=null) {
 
     def jenkins_labels = ['aws-eu-west-1': 'aws-sct-builders-eu-west-1',
                           'aws-us-east-1' : 'aws-sct-builders-us-east-1',
-                          'gce': 'gce-sct-builders']
+                          'gce': 'gce-sct-builders',
+                          'docker': 'gce-sct-builders']
```

Comment:
Why docker can't run on all `sct-builders`

Reply:
but less slots to run it.... all those machines are up and running and we pay for all of them :)

Reply:
@dkropachev I don't understand how are they cheaper. The builders are on-demand instances running all the time. Please change to `sct-builders` as Israel recommended